python/src/tests/test_response.py

```diff
@@ -13,9 +13,14 @@ class TestResponse:
     @pytest.fixture(autouse=True)
     def load_correct_definition(self):
         self.request = {
+            "body": {
+                "name": "DifferentialExpression",
+            },
             "experimentId": "random-experiment-id",
             "timeout": "2099-12-31 00:00:00",
             "uuid": "random-uuid",
+            "ETag": "random-etag",
+            "socketId": "random-etag"
```

Comment:
```suggestion
            "socketId": "random-socketId"
```

Reply:
@cosa65 mentioned that the ETag and socketId have the same value so that's why I did this. Is that correct Martin?

Reply:
I had mentioned one value depended on the other one but they are different.

It isn't really very important because both values are decided by the UI but just for more information, the `socketId` is usually `"WorkResponse-{Etag}"` except for the case of clustering, where socketId is `"broadcast"`.